python/ribasim/tests/test_model.py

```diff
@@ -14,8 +14,8 @@ def test_repr(basic):
 
 def test_solver():
     solver = Solver()
-    assert solver.algorithm is None
-    assert solver.saveat is None
+    assert solver.algorithm == "QNDF"  # default
```

Comment:
Nice that these defaults are now in the schema, and therefore visible in Python.